src/main/java/org/dita/dost/invoker/ConversionArguments.java

```diff
@@ -353,16 +353,16 @@ void printUsage() {
                 .subcommands("transtypes")
                 .subcommands("uninstall")
                 .subcommands("version")
-                .arguments("i", "input", "file", "input file")
-                .arguments("f", "format", "name", "output format (transformation type)")
-                .arguments("p", "project", "name", "run project file")
-                .options("r", "resource", "file", "resource file")
-                .options(null, "filter", "files", "filter and flagging files")
-                .options("l", "logfile", "file", "use given file for log")
-                .options("o", "output", "dir", "output directory")
+                .arguments("i", "input", "file", "Input file")
```

Comment:
For the next release, should we extract these strings into a properties resource file? We don't translate them, but would it be easier if they were not in Java code?

Reply:
Yes, that's an excellent idea, thanks! 👍